next-docs/pages/world/account-delegation.mdx

```diff
@@ -1 +1,74 @@
 # Account delegation
+
+_Account delegation_ allows a _delegator_ address to permit a _delegatee_ address to call a `System` on its behalf.
+This enables multiple use cases.
+
+- **Session wallets**.
+  Normally a wallet requires the user to authorize each transaction separately.
+  In the case of a blockchain game, this means having to authorize every move, which is excessive.
+  Using account delegation players can authorize a different wallet, one whose private key is stored on the client, to play to act on their behalf.
+  Because this wallet's private key is stored in the client, rather than a browser extension, the client can decide when asking for authorization is warranted and when it isn't.
+  By making sure this is a _separate_ wallet, we protect the player's main account in the case of vulnerable or malicious game clients.
+
+- **Approvals**.
+  MUD applications can have a feature similar to [the ERC-20 `approve` function](https://eips.ethereum.org/EIPS/eip-20#approve).
+
+## Creating a delegation
+
+First, the delegator has to call [`registerDelegation`](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol#L234-L263).
+This function takes three parameters:
+
+- `delegatee`, the address given the privileges.
+  This can be `address(0)` to give permission to all addresses.
```

Comment:
```suggestion
  This can be `address(0)` to let this delegation apply to all callers. Note that combining `address(0)` with an unlimited delegation is discouraged, as it would allow anyone to perform any action on behalf of the delegator.  
```